data/sql/updates/pending_db_world/rev_1559162254925218831.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1559162254925218831');
+
+DELETE FROM `spell_script_names` WHERE `ScriptName`="spell_pal_item_healing_discount" AND `spell_id`=37705;
```

Comment:
please use single quotes to wrap strings in SQL:

```sql
`ScriptName`='spell_pal_item_healing_discount'
```

Reply:
done!!